app/model/PhoneNumber.scala

```diff
@@ -1,3 +1,18 @@
 package model
 
+import com.okta.sdk.resource.model.User as OktaUser
+
 case class PhoneNumber(countryCode: Int, localNumber: String)
+
+object PhoneNumber {
+  def fromOktaUser(oktaUser: OktaUser): Option[PhoneNumber] =
+    Option(oktaUser.getProfile.getPrimaryPhone).flatMap(fromPhoneNumberString)
+
+  private val phoneNumberPattern = "\\+(\\d{1,2}) (.+)".r
```

Comment:
Is this standardised somewhere? I don't trust phone numbers to be internationally consistent. I know that we use [libphonenumber](https://github.com/google/libphonenumber) in IDAPI (or its Scala port).

Reply:
I think the safest thing would be not try to parse it at all - just take it from the field verbatim.  But I think `/user/me` expects a separate country code and this is the most lax pattern I can think of - to expect a country code followed by some content of any sort.

Reply:
I don't think Okta has any vaildation at all in terms of phone number it's just a string. If we want the phone to be in any sort of format we can't just take whatever clients send.
I guess the question is whether the "legacy" system was wrong to enforce a format for phone numbers or if we are ok with someone entering anything in that field

Reply:
If we can have some meaningful structure it's probably a good thing.  I'll have a look at the libphonenumber library.

Reply:
I forgot this pr was about reading, I guess the validation is not that important on the way out of Okta. 
It might be useful if we want to ignore wrongly formatted phone numbers in user profiles, but the validation is more important when updating the phone number though gatehouse

Reply:
oh I see @raphaelkabo 's point though that we need to be able to separate the country code from the local number reliably here